ocs_ci/ocs/pgsql.py

```diff
@@ -258,6 +275,38 @@ def validate_pgbench_run(self, pgbench_pods):
             all_pgbench_pods_output.append(pg_output)
         return all_pgbench_pods_output
 
+    def get_pgsql_nodes(self):
+        """
+        Get nodes that contain a pgsql app pod
+
+        Returns:
+            list: Cluster node OCP objects
+
+        """
+        pgsql_pod_objs = self.pod_obj.get(selector=constants.PGSQL_APP_LABEL, all_namespaces=True)
+        log.info(f"Create a list of nodes that contain a pgsql app pod (without duplicate nodes in the list)")
+        nodes_set = set()
+        for pod in pgsql_pod_objs['items']:
+            log.info(f"pod {pod['metadata']['name']} located on node {pod['spec']['nodeName']}")
+            nodes_set.add(pod['spec']['nodeName'])
+        return list(nodes_set)
+
+    def respin_pgsql_app_pod(self):
+        """
+        Respin the pgsql app pod
+
+        Returns:
```

Comment:
Here we are waiting for the pod to reach status running and not returning any thing. Do you think this has to be removed?